### ISSUE TITLE
camera view subxsheet

### ISSUE THREAD

**Issue by bobbybutterfly** [opentoonz/opentoonz#2249]
camera view (position, animation) in subxsheet does not match camera view in xsheet

**Comment by shun-iwasawa** [comment]
@bobbybutterfly 
Yes, they usually does not match and it's not a bug.
Collapsing columns does not bring the movement of camera in the parent xsheet into subxsheet.
The parent xsheet may also has extra animations outside of the subxsheet which can alter the appearance of the camera view.
I think changing such behavior and always showing the view from the parent xsheet even when toggling camera view in subxsheet will lead to another confusion.

**Comment by bobbybutterfly** [comment]
@shun-iwasawa 
Ok, thanks, i understand
but then its difficult to coordinate a subxsheet character movement to the camera movement

**Comment by bobbybutterfly** [comment]
No, the other way around! I have a camera animation on my main set and the characters are subxsheets. The cam should follow the movment of characters hand but i cant see what camera1 does, while i am in the subxsheet

**Comment by gab3d** [comment]
@bobbybutterfly 
have you tried using the Toggle Edit In Place option from the Xsheet menu? (or the rightmost button in the Xsheet Toolbar)

That should allow you to be inside de sub-xsheet while still viewing the result as if tou were in the main xsheet.

**Comment by bobbybutterfly** [comment]
I tried this. Does not work for the camera

**Comment by phillycheese007** [comment]
shun-iwasawa, when you say "The parent xsheet may also has extra animations outside of the subxsheet which can alter the appearance of the camera view." do u mean other animated elements effecting camera motion, FX, ...?   If i am understanding you correctly if these things are part of the master scene(the top most parent xsheet) presumably you would expect to see all of that when using edit in place?  now i find myself wondering why the camera is even included when you create a sub-x sheet...only to hold its data if you decide to explode it?  I was under the impression that most people would use the master xsheet to set the scene and such...are some people using the sub-x sheets as self contained scenes each with their own camera settings/movements?

I had not even gotten far enough in an animation to notice this.....this does make "edit in place" seem  only half true. Perhaps a "toggle in place" right click option... "levels, camera, levels and camera".  The "toggle in place" button could be highlighted in different colors for each different option.  Also the viewport camera outline would need to say which x-sheet camera is being used.

I am very curious if anyone has any examples of why they would not want to include the master x sheet camera when using "edit in place" in a sub-x sheet.

**Comment by bobbybutterfly** [comment]
@shun-iwasawa 
Maybe it would help, if one could choose main Xsheet as the only one for camera movements and kill the camera in all subXsheets. Then the cam movements could be visible in the subXsheets and one could adjust his walking character there to the camera
